plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray.yaml

```diff
@@ -0,0 +1,162 @@
+hydra:
+  launcher:
+    cls: hydra_plugins.hydra_ray_launcher.ray_launcher.RayLauncher
+    params:
+      # mode can be: local or aws
+      ray_init_config: # ray.init(**ray_init_config)
+        address: auto
+      ray_remote_config: # @ray.remote(**ray_remote_config)
+        num_cpus: 1
+        num_gpus: 0
+      mode: local
+      modes:
+        local:
```

Comment:
1. local is really empty?
2. this looks like a syntax error (I believe it's not). if you want to indicate an empty map you can do:
```yaml
local: {}
```